### ISSUE TITLE
Read more buttons only appear on second load

### ISSUE THREAD

**Issue by kevinmarks** [aaronpk/Monocle#26]
My [Comics channel](https://monocle.p3k.io/channel/DfPkazfdLf8Cvir8Y3RB1NC3) shows this well - on first load it crops the images but the "read more" doesn't show. If I click another channel and back, the buttons appear.

**Comment by aaronpk** [comment]
Yep thanks, this has happened to me too. I need to trigger the bit of JS that enables the "read more" link after the image has loaded, since it calculates the height of the entry to determine whether to show the link.
